resources/views/volumes/clone.blade.php

```diff
@@ -0,0 +1,195 @@
+
+@push('scripts')
+    <script type="text/javascript">
+        biigle.$declare('destinationProjects', {!!$destinationProjects!!});
+        biigle.$declare('volume', {!!$volume!!});
+        biigle.$declare('name', '{!!old('name',$volume->name)!!}');
+        biigle.$declare('fileLabelTrees', '{!!$labelTrees!!}');
+        biigle.$declare('selectedFilesIds', '{!! collect(old('only_files',[])) !!}');
+        biigle.$declare('selectedFiles', '{!! collect(old('selected_files',[])) !!}');
+        biigle.$declare('fileLabelIds', '{!! collect(old('only_file_labels',[])) !!}');
+        biigle.$declare('annotationLabelTrees', '{!!$labelTrees!!}');
+        biigle.$declare('cloneFileLabels', {{old('clone_file_labels',false)}});
+        biigle.$declare('cloneAnnotations', {{old('clone_annotations',false)}});
+        biigle.$declare('annotationLabelIds', '{!! collect(old('only_annotation_labels', [])) !!}');
+        biigle.$declare('cloneUrlTemplate', "{{ url("api/v1/volumes/{$volume->id}/clone-to/:pid") }}")
+    </script>
+@endpush
+
+@section('content')
+    <div class="container">
+        <div class="col-sm-8 col-sm-offset-2 col-lg-6 col-lg-offset-3">
+            <div class="form-group">
+                <h2 class="row">Clone volume "{{$volume->name}}"</h2>
+                <br>
+            </div>
+            <form id="clone-volume-form" class="clearfix" role="form" method="POST" :action="getCloneUrl"
+                  enctype="multipart/form-data" v-on:submit="startLoading" v-on:keydown.enter.prevent>
+                <div class="row">
+                    <div class="form-group">
+                        <label>New volume name</label>
+                        <input type="text" class="form-control" name="name" id="name" v-model="name"
+                               placeholder="My new volume name" ref="nameInput" value="{{old('name')}}" required
+                               autofocus minlength="1" maxlength="512">
+                    </div>
+                    <div class="form-group{{ $errors->has('name') ? ' has-error' : '' }}">
+                        @if($errors->has('name'))
+                            <span class="help-block">{{ $errors->first('name') }}</span>
+                        @endif
+                    </div>
+
+                    <div class="form-group">
+                        <label>New volume destination project</label>
+                        <typeahead class="typeahead--block" :items="getProjects"
+                                   placeholder="Select destination project"
+                                   title="Select project to clone volume to"
+                                   v-on:select="setProject" :clear-on-select="false"
+                                   :value="setDefaultProject" required></typeahead>
+                    </div>
+
+                    <div class="checkbox">
+                        <label><input type="checkbox" id="files" v-model="cloneFiles">
```

Comment:
You should add help text blocks for all of the checkboxes. See the reports form for an [example](https://github.com/biigle/reports/blob/5f842b59f29974e010bc03e14c0d84a2eb8aaed4/src/resources/views/volumeReports.blade.php#L105-L107).

When "filter images" is checked, you should also display a text that explains how the "*" wildcard can be used ([example text](https://github.com/biigle/core/blob/3ec7819bbadc889895c2eb5561b05520bafd6c8b/resources/assets/js/volumes/stores/filters.js#L177)).